bundle/README.md

```diff
@@ -256,6 +256,12 @@ To install Prometheus you will need to follow this steps:
         ```sh
         $ oc get route kiali -o jsonpath='{.spec.host}' -n istio-system
         ```
+## Validating the configuration
+
+The Operator validates the contents of the `spec.values` parameter. When parsing parameter contents, if the Operator encounters an unknown value, it generates an error. To force the Operator to pass an unknown value, use the `spec.rawValues` field.
```

Comment:
replace `parameter` with something like `field`.

`spec.values` is a field. When operator parses this field and encounters an unknown value, it will trigger an error.

Reply:
Done